content/docs/0.14.x/reference/miscellaneous/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Miscellany
```

Comment:
Have never seen this word in docs before but looks legit

Reply:
I'm not in love with this term.  "Miscellaneous Information" is an alternative but it's so long...  It's back to the old question about what to call man.5 in fhe Unix man pages...